postgres-docker/backup.sh

```diff
@@ -1,7 +1,13 @@
 #!/bin/bash
```

Comment:
I think there's an issue with how this file is updated when changing it. Given it's copied to the image from the Dockerfile to `/backups`, but later when starting a container with this image the `/backups` path is put on a volume. So when updating an existing setup where `/backups` is already a volume path, the content of this file doesn't get updated.

Reply:
Should we put this script into a path that isn't on a volume to ensure that it can get updated with newer versions of the docker image?

Reply:
That seems good to me!

Reply:
Moved both scripts out of the volume location and to `/usr/local/bin`, this way we can easily update them in the future.